requirements.txt

```diff
@@ -0,0 +1,5 @@
+Django==2.2.0
```

Comment:
Check if there are more up-to-date patch versions to avoid having security issues: [django downloads](https://www.djangoproject.com/download/)